app/jobs/delete_pii_from_old_claims_job.rb

```diff
@@ -0,0 +1,11 @@
+# Runs weekly at 00:30 on a Sunday morning, and requests that eligible claims
+# (rejected or paid, and more than 60 days since their last update) have their
```

Comment:
This comment description now talks about behaviour that the job has no responsibility or knowledge of, i.e. what gets scrubbed. Let's move that info to the file for `Claim::Cleaner`.